src/Layout.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Outlet } from "react-router-dom";
+
+import Navbar from "./container/Navbar/Navbar";
+
+const Layout = () => {
+  return (
+    <>
+      <Navbar />
+      <Outlet />
+    </>
+  );
```

Comment:
You should check if there is a need for any properly styled `div` instead of React Fragment